lib/translations.json

```diff
@@ -5218,6 +5222,10 @@
       "key": "Trial/Form/initial/beforeSignIn",
       "value": "Melden Sie sich an und lesen Sie die ganze Serie. "
     },
+    {
+      "key": "Trial/Form/initial/afterSignIn",
+      "value": "Sie können sich in den kommenden Tagen ohne Einschränkung bei uns umsehen."
```

Comment:
hast du den Satz gegenlesen lassen? sonst gerne kurz in #redaktion_produktion posten für double check

Reply:
Er kommt so 1:1 an anderen stellen vor. Hab ih von einem anderen Form geklaut. Ich schicks noch rum aber ist glaub nicht merge blocking.